STYLE_GUIDE.md

```diff
@@ -0,0 +1,461 @@
+# Pony Standard Library Style Guide
+
+This is a document describing the preferred practices for writing code for the Pony standard library. Other projects are free to use the same style, but you are free to use any style in other projects. The following conventions are meant to ensure consistency and readability of Pony code. Any of the following rules may be broken in situations where the recommended practice would diminish readability.
+
+
+## Formatting
+
+### Line Length
+
+The maximum line length is 80 columns.
```

Comment:
I do agree that a modest max column limit makes the code easier to read (less blank area) and easier to compare side by side. But do you really have displays that you use for this that can't show 100x2 columns?

I just wanted to get my 2¢ in before it was too late, but I'll stop whining if no one else prefers 100.